### ISSUE TITLE
wwe games patch plz

### ISSUE THREAD

**Issue by Wolftine45** [xenia-canary/game-patches#324]
wwe legand of wrestelmania ,wwe 12/13/ wwe 2k 14 patch plz!

**Comment by Milkvich11** [comment]
I agree, wwe 2k14 is desperately needs a patch

**Comment by Milkvich11** [comment]
I agree, wwe 2k14 is desperately needs a patch. It's either crashes with d3d12, or runs super slow with vulkan. With vulkan its also misses character models
